### PR TITLE
New package: python3-owslib-0.25.0

### PR DESCRIPTION
**Pull request by gabriel-russo**
#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86-LIBC)

### CODE REVIEW

**Comment by gabriel-russo** [comment]
commit https://github.com/void-linux/void-packages/pull/27041/commits/36d98fcc331307c6a493ea9d7dc33c8407baeddc and commit https://github.com/void-linux/void-packages/pull/29398/commits/2cc879c0e351200c86bfd7cfcf2ac2c02ca2ec24 are templates of this package  but discontinued, i'll give it a try building it, waiting for approval.

**Comment by gabriel-russo** [comment]
@paper42 This package is ready to be merged

File: srcpkgs/python3-owslib/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-owslib'
+pkgname=python3-owslib
+version=0.25.0
+revision=1
+wrksrc="OWSLib-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-pip"
```
Comment (paper42) [inline srcpkgs/python3-owslib/template:7]: is pip really required?

**Review by paper42 — COMMENTED** [review]
If I understand this package is a library and it doesn't make sense to have a library that isn't used by any other packages. Is there a package that you want to package, but depends on library? If so, the package and all its dependencies should be in one PR.

**Comment by paper42** [comment]
also, please make sure the commit email is added to your github profile to properly attribute the commit to you in gh UI

**Comment by gabriel-russo** [comment]
Sorry, i made a mistake in git! All commits from master was merged on my fork and added here in this PR.

**Pull request merged**